WordPress/src/main/res/drawable-night/img_illustration_empty_results_162dp.xml

```diff
@@ -0,0 +1,31 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="196dp"
+    android:height="162dp"
+    android:viewportWidth="196"
+    android:viewportHeight="162">
+  <path
+      android:pathData="M98.5,88.5m-60.5,0a60.5,60.5 0,1 1,121 0a60.5,60.5 0,1 1,-121 0"
+      android:fillColor="#F1F8FF"
```

Comment:
We might want to consider replacing these colors with references to the colors.xml, wdyt? If these colors don't exist in the colors.xml I think we might want to discuss it with the designers - I believe the colors.xml should contain our complete color pallete and afaik all the illustrations should use only colors from this pallete. But I'm not 100% sure about this.

Reply:
@mbshakti 
> If these colors don't exist in the colors.xml I think we might want to discuss it with the designers

I was able to switch out only the white color, but the other colors don't exist within the app yet. Can you provide direction? Thanks.

Reply:
Colors update in commit 2638d9b
@malinajirka - changed to one vector drawable and using colors.xml